src/engine/enginepregain.cpp

```diff
@@ -33,7 +33,9 @@ ControlObject* EnginePregain::s_pEnableReplayGain = NULL;
    A pregaincontrol is ... a pregain.
```

Comment:
Remove this comment and put this text in the header file: "The pregain control alters the volume of the track based on several inputs, including user pregain adjustment, ReplayGain value, and vinyl-like adjustments in volume relative to playback speed."  I think that's all of the cases